lib/requests/login-request.js

```diff
@@ -164,37 +164,42 @@ class LoginRequest extends AuthRequest {
    * @return {string}
    */
   postLoginUrl (validUser) {
-    if (this.clientId) {
+    let uri
+
+    if (this.authQueryParams['client_id']) {
```

Comment:
Why the change from `this.clientId` to `this.authQueryParams['client_id']` here? Was `clientId` not being properly initialized?

Reply:
Fixed